elements/pfe-cta/src/pfe-cta.scss

```diff
@@ -70,95 +98,91 @@ $variables: (
-  fill:    #{pfe-local(Color)};
-  position: relative;
-  top:    .1rem;
-  width:  13px;
-  height: 13px;
-  transition: padding #{pfe-var(animation-speed)} #{pfe-var(animation-timing)};
+:host([aria-disabled="true"]) ::slotted(*) {
+  cursor: default !important;
 }
 
-  ::slotted(:hover) {
-    --pfe-cta__arrow--Padding:     0 0 0 .5rem;
-
-    color: #{pfe-local(Color--hover)} !important;
-    text-decoration: #{pfe-local(TextDecoration--hover)} !important;
-  }
-
-::slotted(:focus) {
-  outline: none !important;
-  color:   #{pfe-local(Color--focus)} !important;
-  text-decoration: #{pfe-local(TextDecoration--focus)} !important;
-
-  &::before {
-    border: 1px solid #{pfe-local($cssvar: BorderColor--focus, $region: inner)};
-  }
+.pfe-cta {
+  &--wrapper {
+    :host([pfe-priority]) &,
+    :host([aria-disabled="true"]) & {
+      display: flex;
+      flex-flow: row nowrap;
+      align-items: center;
+      height: 100%;      
+    }
 
-  .pfe-cta--arrow {
-    fill: #{pfe-local(Color--focus)};
+    @include browser-query(edge) {
+      button,
+      input {
+        @extend %reset-button;
+      }
+    }
   }
-}
-
-
+  &--inner {  // inner border on priority CTAs
+    display: block;
+    height: calc(100% - 4px);
+    width: calc(100% - 4px);
 
-:host(::after) {
-  margin-left: #{pfe-local($cssvar: MarginLeft, $region: arrow)};
-}
+    box-sizing: border-box;
+    position:   absolute;
+    top:    2px;
+    left:   2px;
+    z-index: -1;
 
-:host(:hover) {
-  --pfe-cta__arrow--Padding:     0 0 0 .5rem;
-  background-color: #{pfe-local(BackgroundColor--hover)};
-  border-color:     #{pfe-local(BorderColor--hover)};
+    border: #{pfe-var(surface--border-width)} #{pfe-var(surface--border-style)} #{pfe-local($cssvar: BorderColor, $region: inner)};
+    border-radius: 2px;
+    outline: none;
 
-  ::slotted(*) {
-    color:           #{pfe-local(Color--hover)} !important;
-    text-decoration: #{pfe-local(TextDecoration--hover)} !important;
-    &::before  {
-      border: #{pfe-var(surface--border-width)} #{pfe-var(surface--border-style)} #{pfe-local($cssvar: BorderColor--hover, $region: inner)};
+    :host(:hover) & {
+      border-color: #{pfe-local($cssvar: BorderColor--hover, $region: inner)};
     }
-  }
 
-  .pfe-cta--arrow {
-    fill: #{pfe-local(Color--hover)};
+    :host(.focus-within) & {
+      border-color: #{pfe-local($cssvar: BorderColor--focus, $region: inner)};
+    }
   }
-}
-
-:host(.focus-within) {
-  outline: none !important;
-  background-color: #{pfe-local(BackgroundColor--focus)};
-  border-color:     #{pfe-local(BorderColor--focus)};
+  &--arrow {
+    display: #{pfe-local($cssvar: Display, $region: arrow)};
+    padding: #{pfe-local($cssvar: Padding, $region: arrow)};
+    fill:    #{pfe-local(Color)};
+    position: relative;
+    top:    .3em;
+    width:  #{pfe-local($cssvar: size, $region: arrow)};
+    height: #{pfe-local($cssvar: size, $region: arrow)};
```

Comment:
```suggestion
  vertical-align: middle;
  width:  13px;	  top:    -1px;
  height: 13px;	  width: 0.9em;
  height: 0.9em;
  height: 0.9em;
```

Reply:
Where did `top: -1px` come from? I think -2px looks a little more centered

Reply:
When testing Cassondra's width and height variables looked good so I kept that as is. I also didn't see a difference with `vertical-align:middle` so I didn't include that in the code.